Add Book Dash reports from 2021 and 2022 (#3082)
alan-turing-institute/the-turing-way: workshops/book-dash/2022-May-BookDash-7.md

We greatly appreciate the work that our attendees have accomplished in the project during this short event and thank them for their feedback, a few of which have been highlighted below.

* Well organised - schedules, HackMD documents
* A good split of work/social time
* Friendly and welcoming. Always a positive attitude
* Enough flexibility to account for time zones, work commitments, life commitments (for me at least)
* Very thoughful to have provided accessibility grants to the participants
* 
* A lot of opportunities to work on different types of tasks to improve the TW, flexible schedule to accommodate people with different schedules
* Friendly environment, I felt able/encouraged to try out different things and experiment a bit, while at the same time I did accomplish something!
* The hub was very nice: it was really great to sit and work together and meet others in the community I hadn't met before.
* Liked the reminders to take breaks but having the option to have people to hold me accountable
* 

Delta
---

* Some of the templates were inconsistent with timings and that caused confusion 

> * A way to keep track of all the different documents. Maybe one master HackMD/document which links to all the other HackMDs or important Slack info? (Like a password manager) So we can find the other documents or illustrations easily

Reviewer: ```suggestion
* A way to keep track of all the different documents. Maybe one central HackMD/document which links to all the other HackMDs or important Slack info? (Like a password manager) So we can find the other documents or illustrations easily
```

- Host one of these calls, see details here: https://the-turing-way.netlify.app/community-handbook/coworking.html
- Attend a fireside chat, or speak at one, or organise one in your community: https://hackmd.io/@turingway/fireside-chats
- Contribute new topics or review open pull requests
- Join us on social media platforms or connect in other ways described here: https://hackmd.io/@turingway/demo-intro

### Still wondering where to start?

Here are a few suggestions for good first contributions:

-   Fix a typo in [the book](https://the-turing-way.netlify.com) and/or [repository](https://github.com/alan-turing-institute/the-turing-way).
-   Flag the unclear or duplicated information in the book by opening an [issue on the GitHub](https://github.com/alan-turing-institute/the-turing-way/issues).
-   Fix an [old issue](https://github.com/alan-turing-institute/the-turing-way/issues) or comment on the the [pull requests](https://github.com/alan-turing-institute/the-turing-way/pulls).
-   Write your personal experience of reproducible research in this [google form](https://goo.gl/forms/akFqZEIy2kxAjfZW2).

> -   See more info on the GitHub: [https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/online-collaboration-cafe.md](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/online-collaboration-cafe.md)

Reviewer: ```suggestion
-   See more info on the GitHub: [https://github.com/alan-turing-institute/the-turing-way/blob/main/book/website/community-handbook/coworking/coworking-collabcafe.md](https://github.com/alan-turing-institute/the-turing-way/blob/main/book/website/community-handbook/coworking/coworking-collabcafe.md)
```

    * Application draft template: https://tinyurl.com/tw-bookdash-template
* Eventbrite page for the participants to register: https://www.eventbrite.co.uk/e/the-turing-way-london-book-dash-november-2021-tickets-184864082447.
* Document with all links: https://hackmd.io/@turingway/bookdash-nov2021-index
* Onboarding calls, 02 November: https://hackmd.io/@turingway/bookdash-nov2021-onboarding
* GitHub Skill-up Session and Collaboration Cafe, 03 November: https://hackmd.io/@turingway/bookdash-nov2021-github
* Scriberia Session Bookings: https://hackmd.io/@turingway/bookdash-nov2021-scriberia
* For specific notes from each day, please visit one of the following notes:
    * **Day 1, 08 November**: https://hackmd.io/@turingway/bookdash-nov2021-day1
    * **Day 2, 09 November**: https://hackmd.io/@turingway/bookdash-nov2021-day2
    * **Day 3, 10 November**: https://hackmd.io/@turingway/bookdash-nov2021-day3
    * **Day 4, 11 November**: https://hackmd.io/@turingway/bookdash-nov2021-day4
    * **Day 5, 12 November**: https://hackmd.io/@turingway/bookdash-nov2021-day5
* Post-event feedback: https://hackmd.io/@turingway/bookdash-nov2021-feedback
* Reports from the previous events:

>     * https://github.com/alan-turing-institute/the-turing-way/tree/master/workshops/book-dash

Reviewer: ```suggestion
    * https://github.com/alan-turing-institute/the-turing-way/tree/main/workshops/book-dash
```
